### ISSUE TITLE
Blocked on copyright grounds.

### ISSUE THREAD

**Issue by AdamDanischewski** [rg3/youtube-dl#7488]
I noticed an error with some videos stating blocked on copyright grounds.
"YouTube said: This video contains content from #### ##### #####, who has blocked it in your country on copyright grounds."  

If you are using a proxy and its exit node is in an incorrect region you can stop using the proxy. 

If you use anything to encrypt/anonymize your location like tor for all connections and the exit node is outside of the allowable region you may get an error such as that above. 

So don't expect, for instance: $> torify youtube-dl -v -f best --no-cache-dir "https://www.youtube.com/watch?v=YOUTUBEID" 
with an exit node locked in a blocked region to get you your video. 

If you want to play around and see what I'm talking about, you can 
$> sudo apt-get install tor; tor &; torify curl www.ipmango.com/api/myip 2>/dev/random

**Comment by dstftw** [comment]
How is this related to youtube-dl? Managing proxies is solely user's responsibility. For tor you can easily setup desired exit nodes' countries or even particular exit nodes by node name or fingerprint with `ExitNodes` in `torrc`.
Closing since out of scope of youtube-dl.

**Comment by AdamDanischewski** [comment]
Yes Sergey, I realize that it's nothing to do actually with the youtube-dl side but users of youtube-dl may at some point run into this issue, they can now find a possible source of the problem. It's rare to be blocked but if you proxify connections you can forget and not . They may not know what exit node they are on, so at least if they read this they can find out to use ExitNodes in torrc after they find a proper exit node, using something like: 

 ## Find the country of the current Exit Node
curl ipinfo.io/$(torify curl www.ipmango.com/api/myip 2>/dev/random) | grep -i country 

 ## Randomly change the exit node, repeat above until you are on your appropriate country.
kill -HUP $(ps -ef | grep '[/]usr/bin/tor' | awk '{print $2}')
